features/specs/gezag-minderjarige/immigrant.feature

```diff
@@ -0,0 +1,199 @@
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Aart'
+
+  Regel: Het gezag kan worden bepaald voor een minderjarige die een gewone verblijfplaats in het buitenland heeft gehad en gezamenlijk gezag heeft
+    Nederland erkent ook huwelijk van homoparen in het buitenland. Een buitenlands huwelijk heeft dus voor het bepalen van gezag dezelfde status als een huwelijk in Nederland (1:253sa BW).
+
+    Voorbeeld: minderjarige is in het buitenland geboren en heeft één ouder die ten tijde van de geboorte van minderjarige gehuwd was
+      Gegeven 'Gerda' en 'Zoe' zijn 6 jaar geleden gehuwd
+      En persoon 'Bert'
+      * heeft 'Gerda' als ouder
+      * is geboren in Duitsland
+      * is 5 jaar geleden geboren
+      * is 3 jaar geleden geïmmigreerd
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' gezamenlijk gezag met ouder 'Gerda' en derde 'Zoe'
+
+  Regel: Het gezag kan worden bepaald voor een minderjarige die een gewone verblijfplaats in het buitenland heeft gehad en één ouder heeft
+
+    Voorbeeld: minderjarige is in het buitenland geboren en heeft één ouder waarbij geen huwelijk bekend is
+      Gegeven persoon 'Bert'
+      * is geboren in Duitsland
+      * heeft 'Gerda' als ouder
+      En 'Bert' is 10 jaar geleden geïmmigreerd naar Nederland
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' eenhoofdig ouderlijk gezag met ouder 'Gerda'
+
+  Regel: het gezag kan worden bepaald voor een minderjarige waarvan een of beide gezaghebbenden niet staan ingeschreven in de BRP
+    In dat geval weten we niet met zekerheid of er iets is veranderd in de situatie van deze ouder(s). Bijvoorbeeld of een ouder uit de ouderlijke macht is ontzet of de ouder is overleden.
+    Het gezag wordt in dat geval bepaald op basis van de gegevens van de ouders zoals die bekend zijn. Daarbij wordt dan een toelichting geleverd waarin wordt aangegeven dat de persoon gewijzigd gezag kan laten opnemen in het gezagsregister.
+    De situatie dat beide ouders niet in de BRP staan heeft altijd tot gevolg dat er voogdij wordt uitgesproken.
+
+    Voorbeeld: minderjarige is in Nederland geboren en een van de ouders is niet ingeschreven in de BRP
+      Gegeven persoon 'Bert'
+      * is 1 jaar geleden geboren
+      * heeft 'Gerda' als ouder
+      * heeft 'Günther Schmidt' als ouder die niet met burgerservicenummer is ingeschreven in de BRP
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Günther Schmidt' met de toelichting 'gezag is bepaald op basis van onvolledige gegevens van Günther Schmidt. Feiten die bepalend zijn voor het gezag van Günther Schmidt kunnen betrokkenen in het gezagsregister dan wel bij je woongemeente laten inschrijven, zodat zij worden meegenomen in de gezagsbepaling van minderjarige.'
```

Comment:
.....mogelijk? onvolledige gegevens? Misschien nog even goed naar de toelichting kijken.

Toelichting wordt NIET in v2.7.0 meegenomen. In v2.7.0 wordt WEL gezag bepaald voor een minderjarige met een ouder die in het buitenland woont.